Model/User.php

```diff
@@ -190,9 +176,7 @@ public function unserialize($serialized)
             $this->salt,
             $this->usernameCanonical,
             $this->username,
-            $this->expired,
             $this->locked,
-            $this->credentialsExpired,
```

Comment:
you are still breaking unserialization of old data, and so this cannot be merged as is.